content/committees/specification/guide/index.md

```diff
@@ -0,0 +1,80 @@
+
+## Creation Review {#creation}
+
+![Creation Review](JESP_creation-review.png)
+
+* A Jakarta Specification project is created as any other project under the Eclipse Development Process ([EDP](https://www.eclipse.org/projects/dev_process/)).
+* A Jakarta Specification must be approved by the Specification Committee.
+
+### Steps Involved
+
+1. Create a Project Proposal (*).
+2. Submit a Pull Request to the [Jakarta EE Specifications repository](https://github.com/jakartaee/specifications) using the **[creation review template](https://github.com/jakartaee/specifications/blob/master/creation_review_pr_template.md)**.
+3. Send an email to the [Specification Committee](mailto:jakarta.ee-spec@eclipse.org) to request a creation review.
+
+(*) It may be a good idea to socialize the idea with the Specification Committee to solicit support before proceeding with the project creation.
+
+## Plan Review {#plan}
+
+![Plan Review](JESP_plan-review.png)
+
+* Each specification project needs to engage in a Plan Review prior to any extensive development effort in support of a new Specification Version. 
+
+### Steps Involved
+
+1. Submit a Pull Request to the [Jakarta EE Specifications repository](https://github.com/jakartaee/specifications) using the **[plan review template](https://github.com/jakartaee/specifications/blob/master/plan_review_pr_template.md)**.
+2. Send an email to the [Specification Committee](mailto:jakarta.ee-spec@eclipse.org) to request a plan review.
+
+## Progress Review {#progress}
+
+![Progress Review](JESP_progress-review.png)
+
+* A progress review can be initiated by the project when they want to inform the specification committee about their progress, but are not ready for a release yet.
+* A progress review may be requested by the specification committee if no progress has been made in 12 months.
+
+### Steps Involved
+
+1. Submit a Pull Request to the [Jakarta EE Specifications repository](https://github.com/jakartaee/specifications) using the **[progress review template](https://github.com/jakartaee/specifications/blob/master/progress_review_pr_template.md)**.
+2. Send an email to the [Specification Committee](mailto:jakarta.ee-spec@eclipse.org) to request a plan review.
+
+## Release Review {#release}
+
+![Release Review](JESP_release-review.png)
+
+* A final version of a specification cannot be made generally available until after engaging in a successful release review (with corresponding super-majority approval from the specification committee).
+
+### Steps Involved
+
+1. Submit a Pull Request to the [Jakarta EE Specifications repository](https://github.com/jakartaee/specifications) using the **[release review template](https://github.com/jakartaee/specifications/blob/master/pull_request_template.md)**.
+2. Send an email to the [Specification Committee](mailto:jakarta.ee-spec@eclipse.org) to request a release review.
```

Comment:
On each of these sections, we indicate to send an email to the Spec Committee.  But, what does the submitter do after that?  Do they assume success and move on?  No...  In many cases, these reviews require an official ballot, which takes some time (after reviewing).  Is it too restrictive to add a 3rd step to these Reviews that say something like, "3. Wait for a successful xxxx review before proceeding."?